rtf_vis_tool/src/Components/CoordinateGrid.js

```diff
@@ -0,0 +1,80 @@
+/* Component which renders the x,y,z axes along with the grid lines along the xy plane. Used within
+Data_Association_PC.js.
+
+Author: Adi Singh
+*/
+import React from "react";
+
+// Third-Party Package Imports.
+import {Line, Html} from "drei";  // Used to render axes lines and text.
+
+const lowerGridBound = -10;   // Lower grid bound for coordinate axes.
+const upperGridBound = 10;    // Upper grid bound for coordinate axes.
+const htmlSize = 50;          // Size of the x,y,z axes labels.
+
+// Function that creates the grid lines along with xy plane.
+const getGridLines = () => {
+    var coordPairSet = []
+
+    // Add grid lines parallel to x axis.
+    for (var y = lowerGridBound; y <= upperGridBound; y++) {
+      coordPairSet.push([[lowerGridBound, y, 0], [upperGridBound, y, 0]]);
+    }
+    // Add grid lines parallel to y axis.
+    for (var x = lowerGridBound; x <= upperGridBound; x++) {
+      coordPairSet.push([[x, lowerGridBound, 0], [x, upperGridBound, 0]]);
+    }
+
+    var finalLineGridSet = []
+    coordPairSet.map(coordPair => {
+      finalLineGridSet.push(<Line 
+                              points={coordPair} 
+                              color="black" 
+                              position={[0,0,0]} 
+                              lineWidth={0.1}/>);
+    });
+    return finalLineGridSet;
+}
+
+const CoordinateGrid = () => { 
```

Comment:
Docstring?

Reply:
Since `CoordinateGrid` is the main component of the file, I have its docstring at the top of the file. I have done this for all components.